joomla/joomla-cms
# Convert mod_articles_archive to new structure

### Summary of Changes
This PR converts mod_articles_archive to the new structure

### Testing Instructions
- Use Joomla 4.3
- Create an instance of the module Articles - Archived. Note how it works.
- Apply patch
- Delete administrator/cache/autoload_psr4.php
- Refresh the page, make sure the module still displays the same as before.

### Additional comments
A small change to the layout was made. Now will show the amount of articles the month have archived, for example:

**Before patch:**
December, 2022

**After patch:**
December, 2022 (1)

If is not allowed, let me kown and I revert the change
